AyaandEbraheem/style.css

```diff
@@ -0,0 +1,45 @@
+
+.asidnav{
+    height:100%;
+    width:10%;
+    background-color: rgb(87, 84, 84);
+    position:fixed;
+    top:0;
+    left:0;
+    padding-top: 40px;
+    padding-left:40px;
+    padding-right:25px;
+     overflow-x: hidden;
+     text-align:right;
+     font-family: Monospace;
+  }
+  .asidnav h3{
+    color:white;
+    font-size: 22px;
+    margin-bottom: 0;
+    padding-bottom: 0;
+  }
+  .asidnav a{
+    padding:4px 6px 4px 10px;
+    text-decoration: none;
+    font-size: 10px;
+    color:white;
+    border:1px solid;
+    border-color:lightblue;
+    display:inline;
+    margin-right:3px;
+    }
+  .asidnav p{
+   font-size:10px;
+    color:lightblue;
+     padding-top:0;
+     margin-top:0;
+  }
+  
+  main{
+   margin-left:15%;
+  }
+  
+  
+  
+   
```

Comment:
put only one new line at the end of the file
and out new lines between each two sections of the code